src/Nox.Reference.VatNumbers/Extension/VatNumberExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Nox.Reference.VatNumbers.Models;
+
+namespace Nox.Reference.VatNumbers.Extension
+{
+    public static class VATNumberExtensions
+    {
+        public static string NormalizeVatNumber(this IVatNumber vatNumber)
+        {
+            var number = vatNumber.Number.RemoveSpecialCharacthers();
+
+            return number.ToLower().Replace(vatNumber.CountryAlphaCode2.ToLower(), string.Empty);
```

Comment:
number.Replace(vatNumber.CountryAlphaCode2,string.Empty,StringComparison.OrdinalIgnoreCase));

Reply:
hm, the purpose was to also return a string with "ToLower", but let me think if it's better to use upper one